for/wordpress.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: page
+title: Codeception for WordPress
+hero: wordpress_hero.html
+sidebar: |
+
+  ## Codeception Tests
+
+  * Combine **all testing levels** (acceptance, functional, unit)
+  * use WordPress defined functions, constants, classes and methods in any test
+  * can be written in **BDD** format with Gherkin
+
+  ## Reference
+
+  * [WPBrowser modules library](https://github.com/lucatume/wp-browser) 
+  * ["I'd like this" Demo WordPress plugin](https://github.com/lucatume/idlikethis)
+
+---
+
+## Install
+
+Install latest stable Codeception via Composer:
+
+```bash
+composer require codeception/codeception --dev
+```
+
+Install latest stable WPBrowser package via Composer:
+
+```bash
+composer require lucatume/wp-browser --dev
+```
+
+## Setup
+
+To fully use the WordPress specific modules of the WPBrowser suite you need to setup a local WordPress installation; this is in no way different from what's required to locally develop a WordPress theme or plugin.  
+In the examples below I assume that the local WordPress installation root directory is `/var/www/wordpress`, that it responds at the `http://wordpress.dev` URL and that I am developing a plugin called "Acme Plugin".  
+While it's possible to start from a default Codeception configuration the WPBrowser package comes with its own CLI tool (`wpcept`) and its own bootstrap command.  
+Scaffold the tests using the latter
+
+```bash
+wpcept bootstrap
+```
+
+The command creates the `tests` directory and scaffold the `acceptance`, `functional`, `wpunit` and `unit` suites creating a `tests` sub-directory and a configuration file for each.  
+Edit the `codeception.yml` file to match your local WordPress setup and point the modules to the local WordPress installation folder, the local WordPress URL and so on.  
+If you are using the `WPLoader` module in your tests take care to create a dedicated database for it and not to use the same database the `Db` or `WPDb` modules might use.  
+The use of the modules defined in the WPBrowser package is not tied to this bootstrap though so feel free to set up Codeception in any other way.
+
+## WordPress Unit Tests
+Most references to "WordPress unit tests" imply a reference to [PhpUnit](https://phpunit.de/ "PHPUnit – The PHP Testing Framework") powered tests where, but, a local WordPress instance is loaded in the same scope as the tests before the test methods are executed.  
+So "WordPress" as WordPress is loaded before the tests and "unit tests" as the subject under test is a single class or a group of tightly related functions that can be thought about as a unit.  
+By default WPBrowser creates the `wpunit` suite to keep these tests separate from "real" unit tests stored in the `unit` suite.  
+The `wpunit` suite includes the `WPLoader` module: this takes care of loading, installing and configuring a fresh WordPress installation before each test method runs.  
```

Comment:
Ok, I got the point but to me the wording can be improved. Actually that was pretty hard to read with too much unit test and wordpress words in different context.

I think we can avoid this confusion by calling wordpress-unit-tests to be **integration** tests which is actually correct.